doc/ValueExtraction_DE.md

```diff
@@ -22,40 +22,56 @@ Damit der *Smart Appliance Enabler* weiss, dass die Antwort als JSON interpretie
 
 Die Wert-Extraktion mit einem [regulären Ausdruck](http://www.regexe.de/hilfe.jsp) funktioniert immer. Allerdings erschliesst sich deren Formulierung nicht jedem sofort. 
 
-Zum Testen, ob der gewählte reguläre Ausdruck den gewünschten Wert aus der Antwort extrahiert, eignet sich dieser [Java Regex-Tester](https://www.freeformatter.com/java-regex-tester.html).
+Zum Testen, ob der gewählte reguläre Ausdruck den gewünschten Wert aus der Antwort extrahiert, eignet sich die Webseite [RegEx101](https://regex101.com/). Der Vorteil dieses Testers ist, dass er direkt während der Eingabe evaluiert.
 
 Ausser dem regulären Ausdruck benötigt man die Antwort, aus welcher der Wert extrahiert werden soll. Wenn der *Smart Appliance Enabler* bereits mit diesem Gerät kommuniziert, kann dessen Antwort dem Log entnommen werden.
 
+### Beispiel für RegEx-Anwendung auf JSON-Daten 
```

Comment:
Die Unterscheidung wirkt hier künstlich und nicht notwendig, weil in solchen Fällen der JSON-Path verwendet werden sollte, der weiter oben auf dieser Seite vorgestellt wurde. Regex sollte nur verwendet werden, wenn JSON-Path nicht möglich ist, oder nachfolgend zum JSON-Path.

Reply:
Gut, dann werfe ich das Beispiel raus, kein Problem.

Reply:
Redundantes Beispiel entfernt.